_blog/deploying-laravel-to-kubernetes/index.md

```diff
@@ -0,0 +1,289 @@
+```
+Running the following command:
+
+```bash
+kubectl get services
+```
+will show you a list of running services. You can also view the running service under the "Services" navigation menu within the dashboard. A more exciting way to verify this deployment and the service exposure is obviously seeing the running application in the browser 😊
+
+To obtain the URL of the application (service), you can use the following command:
+
+```bash
+minikube service --url=true laravel-kubernetes-demo
+```
+
+which will output the IP address and port number similar to:
+
+```bash
+http://192.168.99.101:31399
+```
+
+or, launch the application directly in the browser:
+
+```bash
+minikube service laravel-kubernetes-demo
+```
+
+__Scaling__
+
+And that is it. You have successfully deployed the application in Kubernetes. It's exciting. But what's the point of doing all of this? Well, you only have one deployment with a single Pod running, provisioned to a Node with the exposed web service. Let's replicate this pod to two more pods acting as load balancers to the web traffic.
+
+So that you understand where you are at this moment, run the following command to get a list of desired and available Pods:
+
+```bash
+kubectl get deployment
+
+NAME                      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+laravel-kubernetes-demo   1         1         1            1           57m
+```
+
+The output will be "1" for each. You want to have three available Pods so let's scale this up:
+
+```bash
+kubectl scale --replicas=3 deployment/laravel-kubernetes-demo
+deployment "laravel-kubernetes-demo" scaled
+```
+
+Done. You have replicated the first Pod to another two, giving you three Pods running this service. Running the `get deployment` command will verify this.
+
+```bash
+kubectl get deployment
+
+NAME                      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+laravel-kubernetes-demo   3         3         3            3           59m
+```
+
+You can also see this in the Dashboard under Pods or in the Service detail screen.
+
+Now the application's deployment services are running on three 
+separate, replicated Pods.
+
+__Ingress__
+
+You've already achieved great things, you deployed the application and scaled the deployment. You have already seen the running application in the browser when pointed to the cluster's (Minikube) IP address and node's port number. Now, you will see how to access the application through an assigned URL as you would do when deploying to the cloud.
+
+Typically a Kubernetes cluster is firewalled from the internet, and the same is true for the Minikube cluster. To use a URL, in Kubernetes, you need Ingress. An Ingress is a set of rules to allow inbound connections to reach a Kubernetes cluster. The Ingress is necessary because, in Kubernetes, resources such as Pods only have IP addresses only routable by and within the cluster. Meaning that they are not accessible or reachable to and from the world outside.
```

Comment:
Not sure what you mean with this: "Typically a Kubernetes cluster is firewalled from the internet, and the same is true for the Minikube cluster."

But I like the following: "To use a URL, in Kubernetes, you need **an** Ingress."

Maybe we just rephrase or drop the first sentence?

Reply:
Okay. I'll drop the sentence because I didn't want to dedicate another paragraph to explain edge routers as it is a little out of context and too much info for the first time users.